x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/rule_types.ts

```diff
@@ -37,6 +37,7 @@ export default function listRuleTypes({ getService }: FtrProviderContext) {
       name: 'Recovered',
     },
     enabled_in_license: true,
+    has_fields_for_a_a_d: false,
```

Comment:
Such a diff highlights a disadvantage such bulk tests. Instead of testing one specific feature it works with the whole data set so any contract changes require changes here. If the test fails it's hard to say what's a real cause. I'd recommend considering partial asserting via `expect.objectContaining` and etc.